boards/pimoroni-servo2040/examples/pimoroni_servo2040_pwm_servo.rs

```diff
@@ -67,10 +67,31 @@ fn main() -> ! {
     const MAX_PULSE: u16 = 2000;
 
     let mut pwm: hal::pwm::Slice<_, _> = pwm_slices.pwm0;
-    pwm.set_ph_correct();
-    // 50Hz
+
+    // 50Hz desired frequency
+    // Rp2040 clock = 125MHz
+    // Top = 65_535, resolution for counter (maximum possible u16 value)
+    // Wrap = Top+1 (number of possible values)
+    // Phase correction multiplier = 2 if phase correction enabled, else 1
+
+    // Divider = rp2040_clock / (Wrap * phase_correction_multiplier * desired_frequency)
+    // Divider = 125,000,000/(65_536 * 1 * 50)
+    // Divider = 38.14639
+    // Divider int = 38
+    // Divider frac = 3 (3/16 = 0.1875, smallest frac greater than desired clock divider).
     pwm.set_div_int(38);
     pwm.set_div_frac(3);
```

Comment:
Is it a good choice to use fractional dividers when using pwm to control a servo?
I know close to nothing about servo motors, but as I understand it they do not really care about the exact frequency as long as it's in the right ballpark. So I wonder if the jitter introduced by the fractional divider is a bigger issue than a slightly different frequency would be. (But then, with a top value of 65535, the clock jitter would probably average out over the length of a pulse, so it doesn't really matter either.)